projects/gigya/src/auth/services/user-authentication/gigya-user-authentication-token.service.ts

```diff
@@ -0,0 +1,58 @@
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable, throwError } from 'rxjs';
+import { UserToken, OccEndpointsService, AuthConfig } from '@spartacus/core';
+import { catchError } from 'rxjs/operators';
+
+@Injectable()
+export class GigyaUserAuthenticationTokenService {
+  constructor(
+    http: HttpClient,
+    config: AuthConfig,
+    // tslint:disable-next-line:unified-signatures
+    occEndpointsService: OccEndpointsService
+  );
+
+  /**
+   * @deprecated since version 1.1
+   * Use constructor(http: HttpClient, config: AuthConfig, occEndpointsService: OccEndpointsService) instead
+   */
+  constructor(http: HttpClient, config: AuthConfig);
```

Comment:
Drop the deprecated version. Make all required services mandatory and keep only one service version.

Reply:
done